styles.css

```diff
@@ -0,0 +1,500 @@
+}
+
+.arrow-button {
+  padding: 20px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  background-color: var(--gray);
+}
+
+.classhover:hover {
+  transform: scale(1.1, 1.1);
+}
+
+.latest-posts-2 {
+  display: flex;
+  margin-top: 15px;
+  cursor: pointer;
+}
+
+.flex-box-4 {
+  text-align: center;
+  width: 500px;
+  height: 600px;
+  background-color: var(--green-gray);
+  border-radius: 3%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.class2 {
+  margin-left: 10px;
+}
+
+.flex-box-5,
+.flex-box-6,
+.flex-box-7 {
+  width: 460px;
+  height: 600px;
+  position: relative;
+}
+
+.middle-box {
+  margin: 0 10px;
+}
+
+.articles-links {
+  width: 200px;
+  font-size: 15px;
+  margin: 40px auto;
+  padding: 0 0  10px 15px;
+  border-bottom: 4px solid rgb(44, 42, 42);
+}
+
+.articles-links a {
+  color: black;
+  cursor: pointer;
+}
+
+.person-weeks {
+  display: flex;
+  background-color: var(--silver);
+  border-radius: 10px;
+}
+
+.imgperson {
+  transform: rotate(-15deg);
+  border-radius: 10px;
+  cursor: pointer;
+  margin: 20px 20px;
+  height: 150px;
+  width: 150px;
+}
+
+.imgperson:hover {
+  transform: rotate(0deg);
+}
+
+.person-weeks div {
+  margin: 20px auto;
+}
+
+.footer-bottom div a {
+  margin: 0 5px;
+}
+
+.writter-div {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(3, 1fr);
+  margin-top: 40px;
+}
+
+.writter-flexbox {
+  display: flex;
+  margin: 5px 7.5px 15px 7.5px;
+  padding: 10px;
+  border-radius: 5px;
+}
+
+.writer-1 {
+  background-color: var(--green-gray);
+}
+
+.writer-2 {
+  background-color: var(--dark-gray);
+}
+
+.writer-3 {
+  background-color: var(--blue-gray);
+}
+
+.img-writter {
+  margin-right: 15px;
+}
+
+.div-writter {
+  height: 200px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+}
+
+.div-guides {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.guide-flexbox,
+.guide-flexbox-4 {
+  width: 500px;
+  margin: 10px 5px;
+  padding: 25px 10px 25px 25px;
+  display: flex;
+  flex-direction: column;
+  background-color: var(--green-gray);
+  border-radius: 15px;
+}
+
+.guide-flexbox {
+  justify-content: space-between;
+  align-self: center;
+}
+
+.guide-flexbox-3 {
+  height: 400px;
+}
+
+.guide-flexbox-4 {
+  height: 600px;
+  margin: 10px 5px 0 5px;
+}
+
+.guide-flexbox-4 > * {
+  margin: 10px 0;
+}
+
+.guide-flexbox button,
+.guide-flexbox-4 button {
+  border-radius: 10px;
+  margin: 0 auto;
+  padding: 10px;
+  outline: none;
+  border: solid 2px black;
+}
+
+footer {
+  background-color: var(--l-gray);
+  width: 100%;
+  grid-area: footer;
+  padding: 0 0 20px 0;
+}
+
+footer h2 {
+  display: flex;
+  justify-content: center;
+  margin: 50px auto 30px auto;
+}
+
+/* footer ul{
+  display: flex;
+  flex-wrap: wrap;
+  width: 80%;
+  margin: 0 auto;
+  justify-content: space-evenly;
+} */
+
+/* footer ul li {
+  margin: 0 15px;
+} */
+
```

Comment:
## Inline comments

Comments: It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /\* `.nav { color: red;}` \*/  
:heavy_check_mark: **comments about code** /\*Navigation \*/

Reply:
It's done.